.gitignore

```diff
@@ -1,5 +1,5 @@
 /target
-crates/*/target
+crates/**/target
```

Comment:
What was this needed for?

Reply:
This is because the crate is nested in `bevy_reflect`, unlike the ECS fail tests which is at the top level. So it's `target` directory isn't included in the previous ignore pattern.

I chose to keep it nested like this to keep the `crates` dir clean

Reply:
Doesn't that mean it gets included when publishing `bevy_reflect`?

Reply:
Oops! I'll fix that when I get home!

Reply:
Do we still need this if it's moved to `crates/`?

Reply:
I left it just for potential future usage (preventing devs from accidentally committing `target`). I’m assuming it'll be useful for when things are moved to `tests/`, but I can remove it if we don't want it. Just figured we might as well haha

Reply:
@alice-i-cecile this one, right? Sorry, I can remove this change tonight if preferred!

Reply:
Yep :) I think we should revert it just to keep this PR clean.